docs/src/integrations/error-handling.md

```diff
@@ -0,0 +1,139 @@
+[DLQ Stage 1](#dlq-stage-1) verschoben.
+
+Ist das Publishing erfolgreich, übernimmt der Integration-Service mit seinem Message Consumer. 
+
+Für den Fall, dass noch außerhalb des Message-Consumers beim Deserialisieren der Message ein Fehler auftritt, ist ein _ErrorHandlingDeserializer_ registriert,
+der mit den Header-Informationen der Message einen Incident erstellt (Näheres dazu siehe unten).
+
+Tritt bei der Verarbeitung einer Message generell ein Fehler auf, kommt zuerst ein [Retry](#retry)-Mechanismus zum Einsatz.
+
+Bei Fehlern in anderen Verarbeitungsschritten innerhalb der unterstützenden Frameworks wird die Message in die 
+für den Consumer konfigurierte [DLQ Stage 1](#dlq-stage-1) verschoben. 
+
+Kommt es zum Fehlerfall innerhalb des Message-Consumers selbst (z.B. auch beim Aufruf eines externen Systems), sollte ein try-catch-Block erwartete sowie unerwartete Exceptions auf oberster Ebene fangen. 
+Ist das nicht gegeben, führt jede Exception zum Anstieg der Nachrichtenanzahl in der DLQ. 
+Tritt ein Fehler auf, muss der Service entscheiden, ob es sich um einen fachlichen oder technischen Fehler handelt, 
+bzw. ob bei der Prozessinstanz ein BPMN-Error oder ein Incident ausgelöst werden soll. 
+Zum Unterschied zwischen BPMN-Error und Incident 
+siehe [business-error-vs-technical-error](https://docs.camunda.io/docs/components/modeler/bpmn/error-events/#business-error-vs-technical-error).
+
+Um die Fehlerbehandlung für Integration-Service so einfach wie möglich zu gestalten, ist in den cloudstream-utils ein IncidentService und ein BpmnErrorService 
+als Bean konfiguriert, den man sich zum Aufruf injecten kann.
+Je nach Entscheidung, um welche Art von Fehler es sich handelt, wird der entspr. Service für die weitere Verarbeitung aufgerufen.
+Nach der Datenaufbereitung sendet der Service eine Message entweder in das BPMN-Error-Topic (siehe [BPMN-Error-Verarbeitung](#bpmn-error-verarbeitung)) 
+oder das Incident-Topic des Connectors (siehe [Incident Verarbeitung](#incident-verarbeitung)).
+
+
+## Retry
+
+Für alle Message-Consumer ist ein Retry-Handler konfiguriert, der bei Fehlern die Verarbeitung wiederholt anstößt, um kurzfristig bestehende Probleme 
+wie z.B. Netzwerkverbindungsprobleme zu umgehen.
+Standardmäßig werden alle Nachrichten dreimal verarbeitet, bevor sie an die DLQ gesendet werden.
+Der Standard kann aber auch umkonfiguriert werden:
+```
+spring.cloud.stream.bindings.<binding-name>.consumer.maxAttempts
+```
+
+
+## Incident-Verarbeitung
+
+Im Connector empfängt ein Consumer Nachrichten aus dem Incident-Topic und erstellt einen Incident für die entspr. Prozessinstanz.
+Dazu ermittelt er aus der Engine alle Activities, die das Event aus dem Message-Header _messageName_ abonniert haben (Event-Subscription) 
+und erstellt auf dieser Activity einen Incident mit dem Typ _integrationError_.
+Beides erfolgt über die Rest-API der Engine.
+
+Unbehandelte Fehler in der Verarbeitung führen zur Weiterleitung der Message in die DLQ. 
```

Comment:
Den Satz verstehe ich in dem Kontext hier nicht. Mit dem Incident wird dann auch die Message in die DLQ gepackt?

Reply:
Hab den Satz umformuliert.